src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IBinaryOperatorExpression.cs

```diff
@@ -898,5 +903,1194 @@ void M(bool a, bool b, bool c, bool d, bool e)
 
             VerifyFlowGraphAndDiagnosticsForTest<BlockSyntax>(source, expectedGraph, expectedDiagnostics);
         }
+
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
+        [Fact]
+        public void LogicalFlow_06()
+        {
+            string source = @"
+using System;
+class C
+{
+    void M(int? a, int b)
+    /*<bind>*/{
+        GetArray()[0] = (a ?? b) + b;
```

Comment:
>GetArray()[0] [](start = 8, length = 13)

I think scenario when an expression ```GetArray()[0]``` is a target of an assignment is not that interesting and creates extra noise if we are targeting testing of binary operators. It is, however, interesting to use it as the first operand of the binary operator.

Reply:
Simplified this for the newly added tests.

---
In reply to: [164227019](https://github.com/dotnet/roslyn/pull/24465#discussion_r164227019) [](ancestors = 164227019)